src/raftstore/coprocessor/properties.rs

```diff
@@ -620,6 +620,10 @@ impl RangeProperties {
             Err(next_idx) => next_idx - 1,
         };
 
+        if start_offset > end_offset {
```

Comment:
s/>/>=

Reply:
when start_offset == end_offset, the result of `drain(start_offset..=end_offset)` is not empty